tests/integration/test_hermes_adopt_existing_install.py

```
@@ -0,0 +1,240 @@
+        existing_service.write_text("[Service]\nExecStart=/usr/bin/hermes gateway\n", encoding="utf-8")
+
+        env.update(
+            {
+                "SEA_HERMES_BINARY_PATH": str(existing_binary),
+                "SEA_HERMES_CONFIG_PATH": str(existing_config),
+                "SEA_HERMES_ENV_PATH": str(existing_env),
+                "SEA_HERMES_SERVICE_PATH": str(existing_service),
+            }
+        )
+
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)